azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - release/*
+
+resources:
+  repositories:
+    - repository: niveristand-custom-device-build-tools
+      type:       github
+      ref:        main
+      endpoint:   nivs-custom-devices
+      name:       ni/niveristand-custom-device-build-tools
+
+stages:
+  - template: azure-templates/stages.yml@niveristand-custom-device-build-tools
+    parameters:
+      lvVersionsToBuild:
+        - version: '2020'
+          bitness: '32bit'
+        - version: '2021'
+          bitness: '64bit'
+        - version: '2023'
+          bitness: '64bit'
+
+      buildSteps:
+        - projectLocation: 'Source\System Monitor Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Configuration Release'
+
+        - projectLocation: 'Source\System Monitor Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpecAllTargets'
+          target: 'All'
+          buildSpec: 'Engine Release'
+
+      releaseVersion: '23.0.0'
+      quarterlyReleaseVersion: '2023 Q1'
```

Comment:
We'll want to think about how we're versioning these and whether we can single-source the mapping between `releaseVersion` and `quarterlyReleaseVersion`. Doesn't have to be this PR, but at some point soon.

Reply:
If we want to single source any release info, we can do so by adding it into the templates in custom-device-build-tools, which all repos get a portion of their configuration from.  It was originally done this way so that each custom device could have its own release information.

Reply:
Addressing in https://github.com/ni/niveristand-custom-device-build-tools/pull/144